### PR TITLE
RELEASE v1.1.2

### PR DESCRIPTION
**Pull request by alinmigea**
ADD a log for the image information processing time
FIX the issue for the image sorting type
DELETE the unneeded logs and made some refactoring

@plentymarkets/team-multichannel-koboldmaki

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Release Notes für Elastic Export Belboon.de
 
+## v1.1.2 (2017-09-28)
+
+### Behoben
+- Es wurde ein Fehler behoben, bei dem Bilder Probleme verursacht wurden.
```
Comment (mariusbippig) [inline meta/documents/changelog_de.md]: Where is the german translation for "loading"?

File: meta/documents/user_guide_en.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ The following table lists details for settings, format settings and recommended
             Merchant_ProductNumber
         </td>
         <td>
-            <b>Required</b><br>>
+            <b>Required</b><br>
             <b>Limitation:</b> only Numbers allowed, max. 20 characters<br>
             <b>Content:</b> The variation id of the variation.
```
Comment (mariusbippig) [inline meta/documents/user_guide_en.md]: Helena has forbid us to set there a "Content:" or in german "Inhalt:". Can you please delete that?

File: resources/lang/de/logs.properties [hunk 3]
```diff
@@ -1,9 +1,3 @@
-variationNotPartOfExportStock       = "Variante ohne Bestand."
-variationNotPartOfExportPrice       = "Variantenpreis negativ."
-
-writtenLines                        = "Anzahl der geschriebenen Zeilen."
-esDuration                          = "Auslesens der Artikeldaten aus Elastic Search."
 occurredElasticSearchErrors         = "Fehler bei Elastic Search."
-buildRowDuration                    = "Datei wurde beschrieben."
-fileGenerationDuration              = "Datei wurde komplett erstellt."
+imageInformationDuration            = "Bildinformationen wurden komplett erstellt."
```
Comment (mariusbippig) [inline resources/lang/de/logs.properties]: Attention to the exact translation!

File: src/Generator/BelboonDE.php [hunk 4]
```diff
@@ -364,7 +334,7 @@ private function getShipping($variation):string
             $shippingCost = $this->shippingCostCache[$variation['data']['item']['id']];
         }
 
-        if(!is_null($shippingCost))
+        if(!is_null($shippingCost) && $shippingCost > 0)
```
Comment (mariusbippig) [inline src/Generator/BelboonDE.php:337]: shipping costs can be zero

**Review by mariusbippig — CHANGES_REQUESTED** [review]

File: src/Generator/BelboonDE.php (hunk 4 above)
Comment (alinmigea) [reply src/Generator/BelboonDE.php:337]: from the belboon documentation the only condition is to be a `varchar(255)`, there aren't any restriction to `empty` string or formatted string `0.00`

File: meta/documents/changelog_en.md [hunk 5]
```diff
@@ -1,9 +1,15 @@
 # Release Notes for Elastic Export Belboon.de
 
+## v1.1.2 (2017-09-28)
+
+### Fixed
+- An issue was fixed which caused images loading problems.
```
Comment (mariusbippig) [inline meta/documents/changelog_en.md]: An issue was fixed which caused images not correctly to be exported.

File: meta/documents/changelog_de.md (hunk 1 above)
Comment (mariusbippig) [reply meta/documents/changelog_de.md]: Es wurde ein Fehler behoben, durch den Bilder nicht korrekt exportiert wurden.

**Review by mariusbippig — CHANGES_REQUESTED** [review]

File: meta/documents/changelog_en.md (hunk 5 above)
Comment (alinmigea) [reply meta/documents/changelog_en.md]: updated

**Review by mariusbippig — APPROVED** [review]

**Pull request merged**